react/src/components/Browse.js

```diff
@@ -24,7 +23,7 @@ class Browse extends Component {
     this.updateUserList()
   }
 
-  updateUserList() {
+  updateUserList = () => {
```

Comment:
eri funktiotyyli, miksi? (Rasmus ei antanu munkaan tehä näitä :D)

Reply:
Korkki käski tehä tommosen : D

Reply:
Okei :D en mä näistä oikeesti mitään tiiä